json/Card/web-of-lies.json

```diff
@@ -0,0 +1,27 @@
+[
+	{
+		"clan": "scorpion",
+		"cost": null,
+		"deck_limit": 1,
+		"element": "water",
+		"fate": null,
+		"glory": null,
+		"honor": null,
+		"id": "web-of-lies",
+		"influence_cost": null,
+		"influence_pool": null,
+		"military": null,
+		"military_bonus": null,
+		"name": "Web of Lies",
+		"political": null,
+		"political_bonus": null,
+		"role_restriction": null,
+		"side": "province",
+		"strength": null,
```

Comment:
It won't work. I need to change the data type first, so it becomes a string. And then I need to change all the other cards' data :(